PowerPointLabs/PowerPointLabs/ColorsLab/ColorsLabPaneWPF.xaml.cs

```diff
@@ -94,6 +94,22 @@ public void LoadFavoriteColorsFromPath(string filePath)
             dataSource.LoadThemeColorsFromFile(filePath);
         }
 
+        public List<Color> GetRecentColorsPanelAsList()
+        {
+            List<HSLColor> recentHslColors = dataSource.GetListOfRecentColors();
+            List<Color> recentColors = new List<Color>();
+            foreach (HSLColor recentHslColor in recentHslColors)
+            {
+                recentColors.Add(recentHslColor);
+            }
+            return recentColors;
+        }
+
+        public void ClearRecentColorsPanel()
```

Comment:
Is this method necessary considering that it just calls `EmptyRecentColorsPanel` ?

Reply:
Not really 😛 Changed!